packages/sdk/client-services/src/packlets/diagnostics/diagnostics.ts

```diff
@@ -179,19 +180,22 @@ const getSpaceStats = async (space: DataSpace): Promise<SpaceStats> => {
         id: credential.id,
       })),
 
-    members: Array.from(space.inner.spaceState.members.values()).map((member) => ({
-      role: member.role,
-      identity: {
-        identityKey: member.key,
-        profile: {
-          displayName: member.assertion.profile?.displayName,
+    members: await Promise.all(
+      Array.from(space.inner.spaceState.members.values()).map(async (member) => ({
+        role: member.role,
+        identity: {
+          did: await createDidFromIdentityKey(member.key),
```

Comment:
also expose your own DID in `halo.identity.get().did` ?

Reply:
yeah, it is 👍